server/Game.js

```diff
@@ -154,21 +172,24 @@ export default class {
     }
   }
 
+  setAvailableWords() {
+    this.availableWords = [...words].filter(word => !this.words.includes(word));
```

Comment:
If we keep `this.words` we don't need to store the result of this in `this.availableWords`.
Instead, let's just inline it in `sendWordCandidates()` below, sth like:
```
const availableWords = words.filter(word => !this.words.has(word));
```